apps/api/app/Domain/Missions/Actions/CreateMission.php

```
@@ -22,8 +22,10 @@ public function handle(User $user, Client $client, CreateMissionData $data): Mis
         $this->validateMission->handle($client, $data);
 
         return DB::transaction(function () use ($user, $client, $data): Mission {
-            if ($data->rate instanceof MoneyData) {
-                AccountCurrency::assertMatchesAccountUnderLock($user->id, $data->rate);
+            foreach ([$data->rate, $data->referenceDailyRate] as $money) {
+                if ($money instanceof MoneyData) {
+                    AccountCurrency::assertMatchesAccountUnderLock($user->id, $money);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Domain\Shared\Validation\AccountCurrency' in method 'handle'. (undefined)

(StaticAccess)